### PR TITLE
Update and fix pt-br translation

### PR DESCRIPTION
**Pull request by fabianski7**
:)

### CODE REVIEW

**Review by Frenzie — APPROVED** [review]

**Comment by Alkarex** [comment]
Thanks @fabianski7 👍

File: app/i18n/pt-br/conf.php [hunk 1]
```diff
@@ -139,20 +139,20 @@
...
+		'add' => 'Adicionar um método de compartilhamento',
+		'blogotext' => 'Blogotext',
+		'diaspora' => 'Diaspora*',
+		'email' => 'Email',
+		'facebook' => 'Facebook',
 		'more_information' => 'Mais informação',
 		'print' => 'Imprimir',
-		'remove' => 'Remove sharing method',	// TODO - Translation
-		'shaarli' => 'Shaarli',	// TODO - Translation
+		'remove' => 'Remover método de compartilhamento'
```
Comment (Alkarex) [inline app/i18n/pt-br/conf.php]:
```suggestion
		'remove' => 'Remover método de compartilhamento',
```

File: app/i18n/pt-br/install.php [hunk 2]
```diff
@@ -60,17 +60,16 @@
...
 			'nok' => 'Não foi possível encontrar JSON (php-json).',
 			'ok' => 'Você tem a extensão JSON.',
 		),
 		'mbstring' => array(
-			'nok' => 'Cannot find the recommended library mbstring for Unicode.',	// TODO - Translation
-			'ok' => 'You have the recommended library mbstring for Unicode.',	// TODO - Translation
-		),
+			'nok' => 'Não foi possível encontrar a biblioteca recomendada para o Unicode (mbstring).',
+			'ok' => 'Você tem a biblioteca recomendada para o Unicode (mbstring).',
 		'minz' => array(
```
Comment (Alkarex) [inline app/i18n/pt-br/install.php]:
```suggestion
			'nok' => 'Não foi possível encontrar a biblioteca recomendada para o Unicode (mbstring).',
			'ok' => 'Você tem a biblioteca recomendada para o Unicode (mbstring).',
		),
		'minz' => array(
```

File: app/i18n/pt-br/sub.php [hunk 3]
```diff
@@ -26,72 +26,71 @@
...
 		'description' => 'Descrição',
 		'empty' => 'Este feed está vazio. Por favor verifique ele ainda é mantido.',
 		'error' => 'Este feed encontra-se com problema. Por favor verifique se ele ainda está disponível e atualize-o.',
 		'filteractions' => array(
-			'help' => 'Write one search filter per line.',	// TODO - Translation
-			'_' => 'Filter actions',	// TODO - Translation
-		),
+			'help' => 'Escreva um filtro de pesquisa por linha.',
+			'_' => 'Ações do filtro',
 		'information' => 'Informações',
```
Comment (Alkarex) [inline app/i18n/pt-br/sub.php]:
```suggestion
			'_' => 'Ações do filtro',
		),
		'information' => 'Informações',
```

File: app/i18n/pt-br/index.php [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 return array(
 	'about' => array(
-		'agpl3' => '<a href="https://www.gnu.org/licenses/agpl-3.0.html">AGPL 3</a>',	// TODO - Translation
+		'agpl3' => '<a href="https://www.gnu.org/licenses/agpl-3.0.html">AGPL 3</a>',
```
Comment (Alkarex) [inline app/i18n/pt-br/index.php]:
```suggestion
		'agpl3' => '<a href="https://www.gnu.org/licenses/agpl-3.0.html">AGPL 3</a>',
```

**Pull request merged**